app/containers/Login/index.js

```diff
@@ -27,7 +27,7 @@ const Login = () => {
   const handleLogin = (email, password) => {
     authenticateUser({ variables: { email, password } })
       .then(({ data }) => {
-        client.resetStore();
+        client.cache.reset();
```

Comment:
this doesn't actually solve the issue, if you delete this line there's no error as well, so the resetStore is causing the error, but we need to use resetStore or clearStore https://www.apollographql.com/docs/react/networking/authentication/#reset-store-on-logout